Duplicate symbols under php7?

Using Homebrew php7 and zookeeper:

```
$ ./configure --with-libzookeeper-dir=/usr/local/Cellar/zookeeper/3.4.9 --with-php-config=/usr/local/Cellar/php70/7.0.11_5/bin/php-config
```

make finishes with:
```
/bin/sh /Users/jmagnuss/repos/mainevent/work/php-zookeeper/libtool --mode=link cc -DPHP_ATOM_INC -I/Users/jmagnuss/repos/mainevent/work/php-zookeeper/include -I/Users/jmagnuss/repos/mainevent/work/php-zookeeper/main -I/Users/jmagnuss/repos/mainevent/work/php-zookeeper -I/usr/local/Cellar/php70/7.0.11_5/include/php -I/usr/local/Cellar/php70/7.0.11_5/include/php/main -I/usr/local/Cellar/php70/7.0.11_5/include/php/TSRM -I/usr/local/Cellar/php70/7.0.11_5/include/php/Zend -I/usr/local/Cellar/php70/7.0.11_5/include/php/ext -I/usr/local/Cellar/php70/7.0.11_5/include/php/ext/date/lib -I/usr/local/Cellar/zookeeper/3.4.9/include/zookeeper  -DHAVE_CONFIG_H  -g -O2   -o zookeeper.la -export-dynamic -avoid-version -prefer-pic -module -rpath /Users/jmagnuss/repos/mainevent/work/php-zookeeper/modules  php_zookeeper.lo zoo_lock.lo php_zookeeper_session.lo php_zookeeper_exceptions.lo -Wl,-rpath,/usr/local/Cellar/zookeeper/3.4.9/lib -L/usr/local/Cellar/zookeeper/3.4.9/lib -lzookeeper_mt
cc ${wl}-flat_namespace ${wl}-undefined ${wl}suppress -o .libs/zookeeper.so -bundle  .libs/php_zookeeper.o .libs/zoo_lock.o .libs/php_zookeeper_session.o .libs/php_zookeeper_exceptions.o  -L/usr/local/Cellar/zookeeper/3.4.9/lib -lzookeeper_mt  -Wl,-rpath -Wl,/usr/local/Cellar/zookeeper/3.4.9/lib
duplicate symbol _zk_optimeout_exception in:
    .libs/php_zookeeper.o
    .libs/php_zookeeper_exceptions.o
duplicate symbol _zk_connection_exception in:
    .libs/php_zookeeper.o
    .libs/php_zookeeper_exceptions.o
duplicate symbol _zk_session_exception in:
    .libs/php_zookeeper.o
    .libs/php_zookeeper_exceptions.o
duplicate symbol _zk_auth_exception in:
    .libs/php_zookeeper.o
    .libs/php_zookeeper_exceptions.o
duplicate symbol _zk_marshalling_exception in:
    .libs/php_zookeeper.o
    .libs/php_zookeeper_exceptions.o
duplicate symbol _zk_base_exception in:
    .libs/php_zookeeper.o
    .libs/php_zookeeper_exceptions.o
duplicate symbol _zk_nonode_exception in:
    .libs/php_zookeeper.o
    .libs/php_zookeeper_exceptions.o
ld: 7 duplicate symbols for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make: *** [zookeeper.la] Error 1
```

Any suggestions?  Thanks!

--- a/php_zookeeper_exceptions.c
+++ b/php_zookeeper_exceptions.c
@@ -25,6 +25,16 @@
 #include "php_zookeeper.h"
 #include "php_zookeeper_exceptions.h"
 
+#include <Zend/zend_exceptions.h>
+
+static zend_class_entry *zk_base_exception;
+static zend_class_entry *zk_optimeout_exception;
+static zend_class_entry *zk_connection_exception;
+static zend_class_entry *zk_marshalling_exception;
+static zend_class_entry *zk_auth_exception;
+static zend_class_entry *zk_session_exception;
+static zend_class_entry *zk_nonode_exception;
+
 void php_zk_register_exceptions(TSRMLS_D)
 {
 	zend_class_entry ce;

--- a/php_zookeeper_exceptions.h
+++ b/php_zookeeper_exceptions.h
@@ -18,16 +18,6 @@
 #ifndef PHP_ZOOKEEPER_EXCEPTIONS
 #define PHP_ZOOKEEPER_EXCEPTIONS
 
-#include <Zend/zend_exceptions.h>
-
-zend_class_entry *zk_base_exception;
-zend_class_entry *zk_optimeout_exception;
-zend_class_entry *zk_connection_exception;
-zend_class_entry *zk_marshalling_exception;
-zend_class_entry *zk_auth_exception;
-zend_class_entry *zk_session_exception;
-zend_class_entry *zk_nonode_exception;
-
 /**
  * register exceptions
  */

